buildroot-external/package/hassio/fetch-container-image.sh

```diff
@@ -19,7 +39,7 @@ image_tag=$(jq -e -r --arg image_json_name "${image_json_name}" \
 	'.[$image_json_name]' < "${version_json}")
 full_image_name="${image_name}:${image_tag}"
 
-image_digest=$(skopeo inspect --retry-times=5 "docker://${full_image_name}" | jq -r '.Digest')
```

Comment:
Is the skopeo retry completely useless? Or maybe worth having it retry twice or so? :thinking:

Reply:
Not sure, I haven't checked the logs in detail to verify if it fails somewhere and skopeo retries. But IMO it makes more sense to have only a single retry mechanism, instead of having two leading up to M*N retries.

Reply:
I guess the only case where it might make sense if the `skopeo` retry is somewhat smarter, e.g. retries individual layers.

In any case, a complete retry is fine by me too. It should be an exception anyways.